subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -1033,10 +1034,14 @@ static void cis_disabled_cb(void *param)
 
 				ll_conn_iso_stream_release(cis);
 				cig->lll.num_cis--;
+
+			} else if (IS_CENTRAL(cig)) {
+				/* Prevent referencing inactive CIS */
+				cis->lll.flushed = 0U;
+				cis->lll.acl_handle = LLL_HANDLE_INVALID;
 			}
 
 			/* CIS is no longer active */
-			cis->lll.active = 0U;
```

Comment:
Not needed. Always set to 0 in 'flush'

Reply:
I think I missed this point. Where is `cis->lll.active` cleared on flush?
The rest looks good to me.

Reply:
Good question!

Reply:
This is done in `cis_tx_lll_flush`, which in my opinion is the correct place.